enterprise/app/trends/percentile_chart.tsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+
+import * as format from "../../../app/format/format";
+import { ResponsiveContainer, ComposedChart, CartesianGrid, XAxis, YAxis, Line, Legend, Tooltip } from "recharts";
+
+interface Props {
+  title: string;
+  data: any[];
+  extractLabel: (datum: any) => string;
+  formatHoverLabel: (datum: any) => string;
+  extractP50: (datum: any) => number;
+  extractP75: (datum: any) => number;
+  extractP90: (datum: any) => number;
+  extractP95: (datum: any) => number;
+  extractP99: (datum: any) => number;
+}
+
+const PercentilesChartTooltip = ({
+  active,
+  payload,
+  labelFormatter,
+  extractP50,
+  extractP75,
+  extractP90,
+  extractP95,
+  extractP99,
+}: any) => {
+  if (active) {
+    let data = payload[0].payload;
+    return (
+      <div className="trend-chart-hover">
+        <div className="trend-chart-hover-label">{labelFormatter(data)}</div>
+        <div className="trend-chart-hover-value">
+          <div>p50: {format.durationSec(extractP50(data) || 0)}</div>
+          <div>p75: {format.durationSec(extractP75(data) || 0)}</div>
+          <div>p90: {format.durationSec(extractP90(data) || 0)}</div>
+          <div>p95: {format.durationSec(extractP95(data) || 0)}</div>
+          <div>p99: {format.durationSec(extractP99(data) || 0)}</div>
```

Comment:
can you use `format.compactDurationSec` for these so that the y-axis tick labels don't wrap onto multiple lines?